components/articles/public/AuthorBlock.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Icon from 'components/common/Icon';
+import Text from 'components/common/Text';
+import Clickable from 'components/common/Clickable';
+import { AuthorShape } from 'utils/customPropTypes';
+
+import InfoIcon from './InfoIcon';
+
+const AuthorClickable = ({ displayName, isOpen, onClick }) => (
```

Comment:
we store ``AuthorClickable`` with ``AuthorBlock`` in one file because we need this component only in this file, am I right?

Reply:
yep